benchmarks/dynamo/dist_util.py

```diff
@@ -25,9 +25,14 @@
 
 
 def setup(rank, world_size):
-    os.environ["MASTER_ADDR"] = "localhost"
-    os.environ["MASTER_PORT"] = "12355"
-    dist.init_process_group("nccl", rank=rank, world_size=world_size)
+    # set defaults in case torchrun isn't used; no idea why the if is needed, but it hangs torchrun otherwise
+    if not os.getenv("MASTER_ADDR"):
```

Comment:
why do u need the check here but not for RANK and WORLD_SIZE?

Reply:
i literally have no idea.  I think i shouldn't need the check, but without it torchrun was hanging on the call to init process group. i printed the env strings before/after my setting them and they were apparently the same..